nitro-protocol/gas-benchmarks/gasResults.json

```diff
@@ -23,28 +23,34 @@
     }
   },
   "ETHexit": {
-    "satp": 141446
+    "satp": 109734
```

Comment:
With the current gas prices, these transactions are fairly cheap!

0.000000001 eth/gwei * 1,500 $/eth  * 10 gwei/gas * 109,734 gas = $1.65